src/gridstack.ts

```diff
@@ -374,6 +374,7 @@ export class GridStack {
     this._updateStyles();
     if (this.opts.column != 12) {

```

Comment:
this probably should be done for 12. not sure why we ever special cased it.

Reply:
I too 😅